src/common-ui/InlineLink/index.jsx

```diff
@@ -56,6 +60,7 @@ InlineLink.propTypes = {
   onClick: PropTypes.func,
   linkHelpText: PropTypes.string,
   linkText: PropTypes.string.isRequired,
+  targetBlank: PropTypes.bool,
```

Comment:
nit: we should add default props for not required props

Reply:
Default props are to deprecated/will be deprecated. We have used default values, see line 27.